src/path/PathItem.Boolean.js

```diff
@@ -801,7 +854,12 @@ PathItem.inject(new function() {
                             }
                         }
                     }
-                    wind = wind || getWinding(pt, curves, dir, true);
+                    var pathId = path.getId();
+                    var curveIndex = curve.getIndex();
+                    var hCollisions = horCurveCollisionsMap[pathId][curveIndex];
+                    var vCollisions = vertCurveCollisionsMap[pathId][curveIndex];
```

Comment:
Nitpick, but: Elsewhere we join such variable definitions with one `var`

Reply:
Jut be as nitpicking as possible, I want this to look as the rest of the paper code.